modules/core/src/test/java/org/apache/ignite/internal/processors/cache/datastructures/IgniteCountDownLatchAbstractSelfTest.java

```diff
@@ -334,6 +339,58 @@ public void testLatchMultinode1() throws Exception {
     /**
      * @throws Exception If failed.
      */
+    public void testLatchBroadcast() throws Exception {
+        Ignite ignite = grid(0);
+        ClusterGroup srvsGrp = ignite.cluster().forServers();
+
+        int numOfSrvs = srvsGrp.nodes().size();
+
+        ignite.destroyCache("testCache");
+        IgniteCache<Object, Object> cache = ignite.createCache("testCache");
+
+        for(ClusterNode node: srvsGrp.nodes())
+            cache.put(String.valueOf(node.id()), 0);
+
+        for(int i=0; i<500; i++) {
```

Comment:
There must be a space between an arithmetic or comparison operator and an operand
for(int i=0; i<500; i++) -> for(int i = 0; i < 500; i++)

Check the coding guidelines and setup IntelliJ IDEA plugin from ignite/idea/ignite_codeStyle.xml